### PR TITLE
add support for env.yaml (fix #2)

### LINKED ISSUE #1: Hello there

### LINKED ISSUE #2: Consider supporting YAML config files
json is just horrible for maintaining any sort of significant configuration data. No comments!

### PR DESCRIPTION
**Pull request by dabutvin**
falls back to env.yaml if env.json does not exist

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+const TestRunner = require('test-runner');
+const a = require('assert');
+const mock = require('mock-fs');
+const runner = new TestRunner();
+
+runner.test('getByEnv', t => {
+    let config = require('../lib/painless-config');
```
Comment (jeffwilcox) [inline test/test.js]: Spacing feels off here ;-0

Comment (dabutvin) [reply test/test.js]: tightened it up a bit.
that was a close one 🙈

**Comment by dabutvin** [comment]
conflicts fixed!

**Comment by jeffwilcox** [comment]
Thanks @dabutvin ... do you think people still want YAML?

I added to the parent `painless-config-resolver` that we're using in some other projects the ability to preload a dotfile `.env` into the unset environment variables first... those seem more consistent now with Docker vs opposed to YAML goodness.

**Comment by dabutvin** [comment]
I certainly don't want any more yaml.
Let's close this and forget it ever happened!

https://noyaml.com/

**Comment by jeffwilcox** [comment]
hahahahah ok... sorry but thanks Dan!? hope you're well.

**Comment by dabutvin** [comment]
no problem. I'm doing well, nice to hear from you 😄

**Pull request closed without merging**